library/opensuse

```diff
@@ -7,12 +7,12 @@ Constraints: !aufs
 
 Tags: 42.1, leap, latest
 GitFetch: refs/heads/openSUSE-42.1
-GitCommit: c8abb4323ef1744bfb9e5d4f127d9076083f0a6e
+GitCommit: 7c5cf56668ffa49adc69c5d844ebf05aa244415e
 
 Tags: 13.2, harlequin
 GitFetch: refs/heads/openSUSE-13.2
-GitCommit: 308dda3268f8512795c32cf0e738d5e668f2d3a1
+GitCommit: c51b8abbb418e3566ff1f2299f3747e394412ace
```

Comment:
This one is failing to fetch, should it be e0563a1cdb4a4aa99df8ed0428b037e6a7cfa105? It looks like the branch was updated after this PR.